Libraries/Lists/ViewabilityHelper.js

```diff
@@ -10,351 +10,6 @@
 
 'use strict';
 
-import type {FrameMetricProps} from './VirtualizedListProps';
+export type * from '@react-native/virtualized-lists/Lists/ViewabilityHelper';
```

Comment:
This is reaching into internals

Reply:
This is reaching the internals because these `ViewabilityHelper` components/type were never exported by `react-native`, should we update `@react-native/virtualized-lists` to export all `ViewabilityHelper` types and components inside your `@react-native/virtualized-lists/index.js`?

Reply:
Yes we could export the types that are needed from the root of the package.